config/config.ini

```diff
@@ -998,6 +998,9 @@ name = Kracekumar Ramaraju
 [http://krys.ca/category/python/feed/atom/1.0/]
 name = Krys Wilken
 
+[http://krzysztofzuraw.com/feeds/python.rss.xml]
+name = Krzysztof Żuraw Personal Blog
```

Comment:
Can you please remove the "Personal Blog" from the name?  Our convention is just to use the author's name for those cases.